core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-content-editor/dot-add-variable/dot-add-variable.component.html

```diff
@@ -0,0 +1,22 @@
+<p-dataView *ngIf="vm$ | async as vm" [value]="vm.variables" layout="list">
+    <ng-template let-variable pTemplate="listItem">
+        <div class="col-12">
+            <div class="dot-add-variable-list__item flex p-2">
+                <div class="dot-add-variable-list__detail flex-1">
+                    <div class="variable-name text-2xl">{{ variable?.name }}</div>
+                    <div class="variable-description text-black-alpha-50">
+                        {{ variable?.fieldTypeLabel }}
+                    </div>
+                </div>
+                <div class="dot-add-variable-list__action">
+                    <button
+                        class="p-button-outlined p-button-secondary"
+                        [label]="'Add' | dm"
+                        (click)="onSave(variable?.variable)"
```

Comment:
`variable?.variable` makes me think it is possible for the variable to be `null` or `empty`. If that is the case, should we disable the button?